public/locales/en.json

```diff
@@ -2,6 +2,7 @@
     "app.header.overview": "Heroes",
     "app.header.dashboard": "My Dashboard",
     "app.button.search": "Search...",
+    "app.button.select": "Select",
```

Comment:
This needs to go into [`build/scripts/translations.ts:11`](https://github.com/yvo-niedrich/goa2-faq/blob/main/build/scripts/translations.ts#L11).

The english locale is (re)-generated during the build and deployment process

Reply:
Added!